libraries/chain/include/eosio/chain/protocol_feature_manager.hpp

```diff
@@ -369,6 +372,9 @@ class protocol_feature_manager {
    vector<builtin_protocol_feature_entry> _builtin_protocol_features;
    size_t                                 _head_of_builtin_activation_list = builtin_protocol_feature_entry::no_previous;
    bool                                   _initialized = false;
+   //default is true and if TLS is not set net_plugin will set it to false
+   //this is done to not be dependent on net_plugin if by any reason node will b started without it
+   bool                                   _security_groups_enabled = true;
```

Comment:
@brianjohnson5972 hypothetical. I didn't test nodeos without `net_plugin` enabled.
Also, `net_plugin` is dependent on `chain_plugin`. So `chain_plugin` always initializes before `net_plugin`.
but since we care of just not sending unencrypted blocks i decided that this is ok. let me test what would be if this will be set to false.